src/Sleep.js

```diff
@@ -65,16 +65,26 @@ class Sleep {
   };
 
   getWinnerNamesFromList(sortedArray, userRepo) {
+    let bestSleepers = this.getBestSleepers(sortedArray);
```

Comment:
May need to add some more tests to account for the helper functions created here